drivers/gpu/drm/virtio/virtgpu_vq.c

```diff
@@ -115,6 +115,22 @@ void virtio_gpu_vblank_ack(struct virtqueue *vq)
 	spin_unlock_irqrestore(&vgdev->vblank[target].vblank.qlock, irqflags);
 	drm_handle_vblank(dev, target);
 
+	spin_lock_irqsave(&dev->event_lock, irqflags);
+	for (i = 0; i < VBLANK_EVENT_CACHE_SIZE; i++) {
+		if (vgdev->cache_event[i]) {
+			if (i < *ret_value) {
```

Comment:
Have we changed the ret_value content? If yes, it is better add some comments for it.